filaments/bambulab.json

```diff
@@ -160,7 +179,26 @@
                 {
                     "name": "Charcoal",
                     "hex": "000000"
-                },
+                }
+            ]
+        },
+        {
+            "name": "{color_name}",
```

Comment:
Here you can remove the Silk prefix from all filaments and put it here like `Silk {color_name}`